packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -15,8 +15,8 @@
 import {createProbot, Probot, ApplicationFunction, Options} from 'probot';
 import {CloudTasksClient} from '@google-cloud/tasks';
 import {v1} from '@google-cloud/secret-manager';
-import {request} from 'gaxios';
 import * as express from 'express';
+import {Octokit} from '@octokit/rest';
```

Comment:
missing dependency?

Reply:
this is a peer dependency in `probot`, so I think we're guaranteed to have it.